bootcamp/javascript/js-fundamentals/loops/readme.md

```diff
@@ -0,0 +1,219 @@
+# Loops and iteration
+
+Looping, for any programming language, is a powerful tool in order to execute a set of instructions, repeatedly, while the expression passed is satisfied. A very basic example can be, to print “Hello World” for 10 times.
+
+![Hello world 10 times](https://raw.githubusercontent.com/CGiustiniani/FrontendLearning/feature/js-loops/bootcamp/javascript/js-fundamentals/loops/bs-hello-world.png)
```

Comment:
this will probably break when your feature branch is merged and after that deleted as the link is pointing to the feature branch iteself

Reply:
Hi, I changed in the link feature/js-loops to main.